### DISCUSSION TITLE
Alpine 3.17 for mcr.microsoft.com/dotnet/aspnet:3.1 docker image

### DISCUSSION THREAD

**Discussion by tvbishan** [dotnet/dotnet-docker#4276] [Q&A] [answered]
Will we receive a docker image with **Alpine 3.17** for **aspnet 3.1** or since the support ended on **December 13, 2022**,  Alpine **3.16** will remain?

**Comment by mthalman** [comment, marked as answer]
Since .NET Core 3.1 support ended on Dec. 13, we will not be providing Alpine 3.17 images for it.
